cmd/ovsdb-mon/shell.go

```diff
@@ -234,18 +234,40 @@ func (s *OvsdbShell) Run(ovsPtr *client.Client, args ...string) {
 			Help:    fmt.Sprintf("%s [Field1 Field2 ...]", name),
 			LongHelp: fmt.Sprintf(
 				"List the content of Table %s", name) +
-				fmt.Sprintf("\n\n%s [Field1 Field2 ...]", name) +
+				fmt.Sprintf("\n\n%s [--filter Field=Value] [Field1 Field2 ...]", name) +
```

Comment:
nice! I think we are also missing a similar update in the output of the --help command:
For example:
```
ovsdb-mon list --help
2021/11/07 23:05:41 libovsdb: trying to connect to DB unix:///run/openvswitch/ovnnb_db.sock
2021/11/07 23:05:41 libovsdb: connected to unix:/run/openvswitch/ovnnb_db.sock
OVSDB Monitoring Shell

List the content of a specific table
Commands:
  ACL                              ACL [Field1 Field2 ...]   <--- pls add --filter text here!
```

Reply:
Done thanks!